### DISCUSSION TITLE
JS and CSS minify

### DISCUSSION THREAD

**Discussion by O-One-Develop** [apache/netbeans#7736] [Q&A]
I can't find in plugin archive nothing that could minify JS and CSS.
I'm sure some time ago there was "JS CSS Minify Compress" because I've installed it in my older computer.
What can I use instead?
I've tried with [https://github.com/Chris2011/minifierbeans] but it's not working on my NetBeans IDE 21.
Someone can help me?
Thankyou

**Comment by Chris2011** [comment]
Hey @O-One-Develop I'm the maintainer of the minification plugin you mentioned. This is the "official" plugin that you used before and you use now. Please be more specific by your words: "it is not working for me". Please visit the repo, use the latest version, you can get it from here: https://github.com/Chris2011/minifierbeans/releases/tag/3.5.1 and if smth is not working as intended, please file a ticket with specific steps to reproduce as possible that I can help you fix the bug. Thx :)
